samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,100 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample allows to build Thread Border Router target on the MCU. 
+It is important to mention that, according to NCP architecture, MCU part
```

Comment:
I think there is a one sentence per line rule in ncs doc: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#rst-file-formatting

applies to most of this doc

Reply:
Done